src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -2305,7 +2305,6 @@ public final void setHideDebugLogs(boolean hideDebugLogs) {
         @POST
         public FormValidation doCheckScanTimeoutDuration(@QueryParameter final Integer value) {
             Jenkins.getInstance().checkPermission(Item.CONFIGURE);
```

Comment:
You're still checking the global `Configure` permission for this one, please add the `@AncestorInPath` annotation and check on the current context.

Reply:
This field is on the global configuration page only. Do we have to add the context check ?

Reply:
I have added the @AncestorInPath annotation and Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);

Reply:
My bad on this one, I didn't check for the moment where they're called, but yeah, field called on global configuration page, only need `Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);`. 
No need for `@AncestorInPath` annotation or Item.Configure permission

Reply:
Also, don't forget to push your commit, otherwise I can't review you changes 😄